Resources.py

```
@@ -1,497 +1,494 @@
+		"NOTSET": logging.NOTSET,
+		"DEBUG": logging.DEBUG,
+		"INFO": logging.INFO,
+		"WARN": logging.WARNING,
+		"WARNING": logging.WARNING,
+		"ERROR": logging.ERROR,
+		"FATAL": logging.CRITICAL,
+		"CRITICAL": logging.CRITICAL,
+		};
+	__instance						= None;
+	__lock:threading.Lock			= threading.Lock();
+	if (os.path.isfile(".\\log.txt")): CompressLogFile();
```

Ruff (0.14.8)

Multiple statements on one line (colon)

(E701)